docs/Usage.md

```
@@ -155,6 +155,38 @@ private void CleverTapInAppNotificationDismissedCallback(string message)
+{
+    CleverTap.PauseSDK();
+}
+
+// Returning to menu / lobby — resume and flush buffered events
+void OnGameplayEnd()
+{
+    CleverTap.ResumeSDK();
+}
+```
+
+#### Pause SDK
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)